### DISCUSSION TITLE
Question 71: Taxonomic hierarchical field "Section" [RDP]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#648] [Q&A]
"I followed your instructions by using Scientific Name=No and adding the Section name to the field. So IRN 763110 Podotremata looks ok now, but when I tried to use it as a parent of superfamily Cyclodorippoidea 763121 the Hierarchy tab was wonky, and the rank for this superfamily shows as Section. But something even weirder happened. I wanted to see what would happen if I removed 763110 Podotremata as the Parent AND from the Section field in the Higher Classification tab, and save it (would it correctly revert the rank to Superfamily?). The Rank said Infrorder!!! "

**Comment by rondlg** [comment]
Response - in the works ZD 56747. Added to next weeks agenda for further discussion

**Comment by rbaquiran** [comment]
Much of Insect's data in Section isn't actually a section as proposed, but a species group or complex.  I'd love to clean these up sooner than later. Could anyone point me in the direction of where to relocate that info so we can move forward?
Thank you much!

**Reply by fmjjones** [reply]
Can you include a link to the reference material for some representative records where these are an issue? I'm wondering if the either the confidence field in catalogue or OtherRank OtherValue fields in Taxonomy module is appropriate and it really depends on how the data are used.
